### DISCUSSION TITLE
Exporting timedelta type to JSON Schema is exporting as a number

### DISCUSSION THREAD

**Discussion by Chuck-Koverse** [pydantic/pydantic#5687] [Question]
I have this simple model class which declares the Duration as type timedelta, but when I export the model to JSON Schema, the the JSON type comes out as a number and not a string.  Previous versions of Pydantic used to export this as a string.  Is there a new setting that I'm not aware of to export to JSON Schema as a string?  It seems specifying timedelta_isoformat would be enough, but it still exports as a number.

```
class Endurance(BaseModel):
      Duration: Optional[timedelta]
      DurationEnd: Optional[datetime]
      class Config:
            json_encoders = {
                  timedelta: timedelta_isoformat
            }
```

```
"Duration": {
                    "type": "number",
                    "format": "time-delta"
                }
```

**Comment by dmontagu** [comment]
For reference, there is an open issue for this: https://github.com/pydantic/pydantic/issues/5034

Do you know what version of pydantic this behaved differently in? It seems in the latest v1 release it was the same.

**Comment by Chuck-Koverse** [comment]
I don't recall.  I created some models and JSON Schemas last year (December-ish maybe) and I haven't revisited the JSON Schema topic until just recently.  Unfortunately I updated my Pydantic a couple times since then.
